chore(rate-limiting): clarify rate-limiting accuracy (#7607)
Kong/docs.konghq.com: app/_hub/kong-inc/rate-limiting-advanced/overview/_index.md

| Strategy    | Pros | Cons   |
| --------- | ---- | ------ |
| `local`   | Minimal performance impact. | Less accurate. Unless there's a consistent-hashing load balancer in front of Kong, it diverges when scaling the number of nodes.

> | `cluster` | Accurate*, no extra components to support. | Each request forces a read and a write on the data store. Therefore, relatively, the biggest performance impact. |

Reviewer: ```suggestion
| `cluster` | Accurate<sup>1</sup>, no extra components to support. | Each request forces a read and a write on the data store. Therefore, relatively, the biggest performance impact. |
```

Author: applied suggestion ✅

> | `redis`   | Accurate*, less performance impact than a `cluster` policy. | Needs a Redis installation. Bigger performance impact than a `local` policy. |

Reviewer: ```suggestion
| `redis`   | Accurate<sup>1</sup>, less performance impact than a `cluster` policy. | Needs a Redis installation. Bigger performance impact than a `local` policy. |
```

Author: applied suggestion ✅

> **only when `sync_rate` option is set to `0` (synchronous behaviour). Please read the config reference for more details.*

Reviewer: ```suggestion
{:.note .no-icon}
> **\[1\]**: Only when `sync_rate` option is set to `0` (synchronous behavior). See the [configuration reference](/hub/kong-inc/rate-limiting-advanced/configuration/#config-sync_rate) for more details.
```

Author: applied suggestion ✅
